bcipy/task/paradigm/vep/calibration.py

```diff
@@ -226,14 +302,15 @@ def init_calibration_display(parameters: Parameters,
     layout = centered(width_pct=0.95, height_pct=0.80)
     box_config = BoxConfiguration(layout, height_pct=0.30)
 
-    stim_props = VEPStimuliProperties(stim_font=parameters['font'],
-                                      stim_pos=box_config.positions,
-                                      stim_height=0.1,
-                                      timing=timing,
-                                      stim_color=colors,
-                                      inquiry=[],
-                                      stim_length=1,
-                                      animation_seconds=parameters['time_vep_animation'])
+    stim_props = VEPStimuliProperties(
+        stim_font=parameters['font'],
+        stim_pos=box_config.positions,
+        stim_height=0.1,
```

Comment:
This may be a good class variable so we can change it with task parameters as needed